src/commands/deploy.js

```diff
@@ -79,7 +78,6 @@ module.exports = new Command("deploy")
     }
   })
   .before(checkValidTargetFilters)
-  .before(checkFunctionsSDKVersion)
```

Comment:
Note: This check was redundant with other functions deploy code. It's now in `deploy/functions/runtimes/node/index.ts:Delegate#validate`